src/createproject.ts

```diff
@@ -0,0 +1,157 @@
+        if (!pkgName)
+            return
+
+        const authors = await vscode.window.showInputBox({ prompt: 'Please enter the authors of the project', placeHolder: "Default uses 'github.name' and 'github.email' from the global Git config." })
+        if (authors === undefined)
+            return
+
+        let host = await vscode.window.showQuickPick(['github.com', 'gitlab.com', 'bitbucket.org', 'Other'], { placeHolder: 'The URL to the code hosting service where the project will reside.' })
+        if (host === 'Other') {
+            host = await vscode.window.showInputBox({ prompt: 'Please enter the URL to the code hosting service.' })
+        }
+        if (host === undefined)
+            return
+
+        // TODO: add gihub.user as placeholder
+        const user = await vscode.window.showInputBox({
+            prompt: 'Please enter your GitHub (or other code hosting service) username.',
+            validateInput: (input) => {
+                if (input === "")
+                    return 'Username cannot be empty.'
+                return undefined
+            }
+        })
+        if (user === undefined)
+            return
+
+        let juliaVersion = await vscode.window.showQuickPick(
+            ['1.0', '1.1', '1.2', '1.3', '1.4', '1.5', '1.6', '1.7', 'Other'],
+            { placeHolder: 'Please select the minimum allowed Julia version.' }
+        )
+        if (juliaVersion === 'Other') {
+            juliaVersion = await vscode.window.showInputBox({ prompt: 'Please enter the minimum allowed Julia version.' })
+        }
+        if (juliaVersion === undefined)
+            return
+
+        const plugins = await vscode.window.showQuickPick(
+            [
+                { label: 'Project File', picked: true },
+                { label: 'Source Directory', picked: true },
+                { label: 'Git', picked: true },
+                { label: 'License', picked: true },
+                { label: 'README', picked: true },
+                { label: 'Tests', picked: true },
+                { label: 'CompatHelper', picked: true },
+                { label: 'TagBot', picked: true },
+                { label: 'AppVeyor', picked: false },
+                { label: 'BlueStyleBadge', picked: false },
+                { label: 'CirrusCI', picked: false },
+                { label: 'Citation', picked: false },
+                { label: 'Codecov', picked: false },
+                { label: 'ColPracBadge', picked: false },
+                { label: 'Coveralls', picked: false },
+                // { label: 'Develop', picked: false },
+                { label: 'Documenter', picked: false },
+                { label: 'DroneCI', picked: false },
+                { label: 'GitHubActions', picked: false },
+                { label: 'GitLabCI', picked: false },
+                { label: 'PkgEvalBadge', picked: false },
+                { label: 'RegisterAction', picked: false },
+                { label: 'TravisCI', picked: false }
```

Comment:
Is there a way to provide more descriptions? This is probably pretty hard to understand for a newcomer.